internal/pkg/cli/validate.go

```diff
@@ -113,6 +119,27 @@ func validateSvcType(val interface{}) error {
 	return fmt.Errorf("invalid service type %s: must be one of %s", svcType, prettify(manifest.ServiceTypes))
 }
 
+func validateJobName(val interface{}) error {
+	if err := basicNameValidation(val); err != nil {
+		return fmt.Errorf("job name %v is invalid: %w", val, err)
+	}
+	return nil
+}
+
+func validateJobSchedule(sched string) error {
+	if err := basicCronValidation(sched); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Maybe we can remove `validateJobSchedule` and replace it with just `validateSchedule`?

The content of this function is the same as writing:
```go
func validateJobScheduled(sched string) error {
    return basicCronValidation(sched)
}
``

Reply:
Ah, yes--thanks!